src/App.js

```diff
@@ -1,26 +1,29 @@
-import logo from './logo.svg';
+import {
+  Routes, Route, Link,
+} from 'react-router-dom';
+import Books from './components/Books';
+import Categories from './components/Categories';
 import './App.css';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          <code>src/App.js</code>
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
+      <nav className="Navigation">
+        <h1 className="Logo">Bookstore CMS</h1>
+        <ul>
+          <li>
+            <Link to="/">BOOKS</Link>
+          </li>
+          <li>
+            <Link to="/categories">Categories</Link>
+          </li>
+        </ul>
+      </nav>
```

Comment:
- [ **OPTIONAL** ] For best practices, I think it would be better if you create a component file named `Navar.js` then add the above code lines inside. After you can import the component in the `App.js` main component.
```JavaScript
 // Exmaple

// src/components/Navbar.js file
import React from 'react';
import { Link } from 'react-router-dom';

const Navbar = () => (
  <nav className="Navigation">
    <h1 className="Logo">Bookstore CMS</h1>
    <ul>
      <li>
        <Link to="/">BOOKS</Link>
      </li>
      <li>
        <Link to="/categories">Categories</Link>
      </li>
    </ul>
  </nav>
);

export default Navbar;

// src/App.js
import { Routes, Route } from 'react-router-dom';
import Books from './components/Books';
import Categories from './components/Categories';
import Navbar from './components/Navbar';
import './App.css';

function App() {
  return (
    <>
      <Navbar />
      <Routes>
        <Route path="/" element={<Books />} />
        <Route path="/categories" element={<Categories />} />
      </Routes>
    </>
  );
}

export default App;
```